litgpt/lora.py

```diff
@@ -122,8 +122,8 @@ def __init__(
 
         # Actual trainable parameters
         if r > 0:
-            self.lora_A = nn.Parameter(torch.zeros((r, in_features)))
-            self.lora_B = nn.Parameter(torch.zeros((out_features, r)))
+            self.lora_A = nn.Parameter(torch.empty((r, in_features)))
+            self.lora_B = nn.Parameter(torch.empty((out_features, r)))
```

Comment:
Clever. Why do you need to initialize now, when it will be done in the `reset_parameters` method.
But that can also be done for the LoRAQKVLinear class.